lc3/README.rst

```diff
@@ -0,0 +1,186 @@
+.. _lc3:
+
+Low Complexity Communication Codec (LC3)
+########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Low Complexity Communication Codec (LC3) is the default software codec for the :ref:`nrf53_audio_app` application, conformant to the `Bluetooth® LE Audio specifications`_ (Bluetooth 5.2, QDID #156294).
+It is used only by this application for encoding and decoding purposes, but can be used with any other audio application developed with the nRF5340 Audio DK.
```

Comment:
Not quite precise. I think it is better to refer to license.txt not to duplicate info

Reply:
Removed line.